pom.xml

```diff
@@ -35,8 +35,39 @@
     </developers>
 
     <properties>
+        <!-- maven properties -->
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <maven.build.timestamp.format>yyyy-MM-dd HH:mm z</maven.build.timestamp.format>
+        <lifecycle-mapping.version>1.0.0</lifecycle-mapping.version>
+
+        <!-- molgenis specific properties -->
+        <molgenis.source.compile.version>1.8</molgenis.source.compile.version>
+        <molgenis.build.timestamp>${maven.build.timestamp}</molgenis.build.timestamp>
+
+        <!-- code quality monitoring properties -->
+        <sonatype.id>sonatype-nexus-staging</sonatype.id>
+        <sonar.jacoco.itReportPath>${project.basedir}/../target/jacoco-it.exec</sonar.jacoco.itReportPath>
+
+        <!-- plugin versions -->
+        <maven-failsafe-plugin.version>2.8.1</maven-failsafe-plugin.version>
```

Comment:
I'm not convinced that moving the version numbers to a property is an improvement. It gives you a better overview of dependency versions, but makes the individual sections less readable and allows for accidental references to this properties in child modules. What do you think?